README.md

```diff
@@ -109,6 +109,17 @@ privileged         true      [*]       RunAsAny    RunAsAny           RunAsAny
 restricted         false     []        MustRunAs   MustRunAsRange     MustRunAs   RunAsAny    <none>     false            [configMap downwardAPI emptyDir persistentVolumeClaim projected secret]
 ```
 
+### Use HTTPS on OpenShift via Ingress
+#### Pre-requisites:
```

Comment:
I believe that "Enable TLS for dashboard access via Ingress" is better title for this.